update docker examples for Varnish3, remove obsolete 'version' yaml (#1963)
plone/documentation: docs/install/containers/examples/traefik-volto-plone-varnish.md

---
myst:
  html_meta:

>     "description": "Basic Plone 6 setup with only one backend, a ZEO server and data being persisted in a Docker volume."

Reviewer: ```suggestion
    "description": "Basic Plone 6 setup with only one backend, a ZEO server, and data being persisted in a Docker volume."
```

>     "property=og:description": "Basic Plone 6 setup with only one backend, a ZEO server and data being persisted in a Docker volume."

Reviewer: ```suggestion
    "property=og:description": "Basic Plone 6 setup with only one backend, a ZEO server, and data being persisted in a Docker volume."
```

    environment:
      SITE: Plone
      PROFILES: "plone.app.caching:with-caching-proxy"
    environment:
      ZEO_ADDRESS: db:8100

>       ZEO_SHARED_BLOB_DIR: on #otherwise the backend will create its own filestorage/blogstorage

Reviewer: ```suggestion
      ZEO_SHARED_BLOB_DIR: on  # otherwise the backend will create its own blob storage
```

    volumes:

>       - data:/data              #the backend and db need access to the same volume

Reviewer: ```suggestion
      - data:/data              # the backend and database need access to the same volume
```

    ports:
      - 8080:8080
    depends_on:
      - db

> #   If the docker container is run with another uid than the uid owning the local filesystem persistent storage,

Reviewer: ```suggestion
#   If the Docker container is run with a UID other than the UID which owns the local file system persistent storage,
```

> #   explicitly make the container run with the correct uid, for example  user: 1000:1000

Reviewer: ```suggestion
#   explicitly make the container run with the correct UID. For example, `user: 1000:1000`.
```

> #   In that case make sure also the 'db' service gets the same 'user: 1000:1000' configuration,

Reviewer: ```suggestion
#   In that case, also make sure that the `db` service gets the same `user: 1000:1000` configuration,
```

> #   as they both need access to the filesystem

Reviewer: ```suggestion
#   as they both need access to the file system.
```
